src/components/Announce/index.jsx

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-redeclare
```

Comment:
Why do we need to disable `no-redeclare` rule here?

Reply:
Honestly I was trying to just push through the linting errors, as it's taken quite a bit longer to produce this minimal change/required functionality of typescript

Reply:
we should probably just remove that `/* global localStorage */` and replace with window.localStorage

Can save for another PR